service/src/main/kotlin/io/provenance/digitalcurrency/consortium/service/PbcService.kt

```diff
@@ -203,4 +211,28 @@ class PbcService(
                 .toAny()
                 .toTxBody()
         ).throwIfFailed("Accept failed")
+
+    private val maxExpiration = OffsetDateTime.of(2031, 1, 1, 0, 0, 0, 0, ZoneOffset.UTC)
```

Comment:
Should this be relative? Maybe just plus years from current date

Reply:
yeah probably better to just do now() + 10 years

Reply:
long live the consortium